### PR TITLE
Changing fonts, fixing one minor typo in footer.tsx

### PR DESCRIPTION
**Pull request by krishnaglick**
## Change description

Changing fonts to match Typography shown [here](https://www.figma.com/file/FSLlDL9yNhpyOCTGVaZE7K/Grouparoo-Moodboard?node-id=5%3A3)

### Development

- [X] Application changes have been tested appropriately

### Impact

- [X] Code follows company security practices and guidelines
- [X] Security impact of change has been considered
- [X] Performance impact of change has been considered
- [X] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [X] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [X] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

**Comment by krishnaglick** [comment]
React used to be very particular about falsy being returned as null. Seems to handle `&&` fine [now](https://reactjs.org/docs/conditional-rendering.html)

File: components/footer.tsx [hunk 1]
```diff
@@ -175,7 +175,7 @@ export default function Footer() {
                   </Link>
                   <br />
                   <Link href="/integrations/sources/redshift">
-                    <a>Snowflake Integration</a>
+                    <a>Redshift Integration</a>
```
Comment (teallarson) [inline components/footer.tsx:178]: nice

**Review by teallarson — CHANGES_REQUESTED** [review]
Nice!  And nice cleanup too.  

One thing I noticed is the weight, kerning, and line heights don't all match between the Figma file and what I see in the preview deployment.  The Figma file shows adjustments to character spacing in a few spots (headings are -1 to -0.44, overline(?) is 2), and it looks like h1s are "extra-bold" and a few of the classes have line heights of 150-170%. (These settings can be seen over in the "text" menu on the right if you select the component in Figma)

I'm not positive what Bootstrap's defaults are but I'm guessing those pieces are the mismatch!

Screen Shot 2021-10-26 at 4 12 06 PM

I also noticed the code block color scheme, but I'm guessing that's part of a forthcoming theme/color update!

**Review by teallarson — APPROVED** [review]
Looks good!
Screen Shot 2021-10-28 at 11 45 16 AM

File: components/home/whoWeAre.tsx [hunk 2]
```diff
@@ -156,50 +156,46 @@ export default function WhoWeAre() {
...
-                          )}
-                        </>
-                      )}
-                    </span>
-                  }
-                />
-              </Col>
-            )
-          )}
+          {teamMembers.map(({ emojis, previously, ...teamMemberProps }) => (
```
Comment (krishnaglick) [inline components/home/whoWeAre.tsx:159]: This diff looks really ugly but is just using a property over an index.

Comment (krishnaglick) [reply components/home/whoWeAre.tsx:159]: Keeping this change based on the [docs](https://reactjs.org/docs/lists-and-keys.html#keys) recommendation: `We don’t recommend using indexes for keys if the order of items may change.`

**Review by andyjih — APPROVED** [review]
👍  lgtm!

**Pull request merged**